### PR TITLE
Iterating on the spec (part 3)

### PR DESCRIPTION
**Pull request by delvedor**
Continuation of #23

Types tested:
- index
  - [x] request
  - [x] response
- create
  - [x] request
  - [x] response
- delete
  - [x] request
  - [x] response 
- update
  - [x] request
  - [x] response
- get
  - [x] request
  - [x] response 
- bulk
  - [x] request
  - [ ] response (`respone.items.update.get` does not exist)
- search
  - [ ] request (almost there!)
  - [ ] response (E_TOO_MANY_ERRORS)

### CODE REVIEW

File: specification/specs/query_dsl/compound/dismax/DisMaxQuery.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class DisMaxQuery {
 	queries: QueryContainer[];
 	tie_breaker: double;
+  boost: float;
```
Comment (Mpdreamz) [inline specification/specs/query_dsl/compound/dismax/DisMaxQuery.ts:4]: `tab` (do we have `.editorconfig` configured correctly?)

File: specification/specs/query_dsl/full_text/multi_match/MultiMatchQuery.ts [hunk 2]
```diff
@@ -17,4 +17,5 @@ class MultiMatchQuery {
 	type: TextQueryType;
 	use_dis_max: boolean;
 	zero_terms_query: ZeroTermsQuery;
+  boost: float;
```
Comment (Mpdreamz) [inline specification/specs/query_dsl/full_text/multi_match/MultiMatchQuery.ts:20]: Do we want to create a base query with `_name` and `boost`?

File: specification/specs/query_dsl/compound/dismax/DisMaxQuery.ts (hunk 1 above)
Comment (delvedor) [reply specification/specs/query_dsl/compound/dismax/DisMaxQuery.ts:4]: We do have. it, but the input spec is in the wrong format.
I'll open a separate pr to address this.

**Review by Mpdreamz — APPROVED** [review]

File: specification/specs/query_dsl/Operator.ts [hunk 3]
```diff
@@ -1,4 +1,6 @@
 enum Operator {
 	and = 0,
-	or = 1
+	or = 1,
+  AND = 2,
```
Comment (Mpdreamz) [inline specification/specs/query_dsl/Operator.ts:4]: Do we need both casings? I think only caps suffices?

Comment (delvedor) [reply specification/specs/query_dsl/Operator.ts:4]: Both are being used in the flight recorder :S

**Pull request merged**